extensions/lifecycle/LifecycleOplogPopulatorUtils.js

```diff
@@ -1,27 +1,59 @@
 const OplogPopulatorUtils = require('../../lib/util/OplogPopulatorUtils');
+const locations = require('../../conf/locationConfig.json');
 
 class LifecycleOplogPopulatorUtils extends OplogPopulatorUtils {
+
     /**
      * Get extension specific MongoDB filter
      * to get buckets that have the lifecycle
      * extension enabled
      * @returns {Object} MongoDB filter
      */
      static getExtensionMongoDBFilter() {
-        // getting buckets with at least one lifecycle
-        // rule enabled
+        const coldLocations = Object.keys(locations).filter(loc => locations[loc].isCold);
```

Comment:
Why not just `locations.filter(loc => loc.isCold);` ?

Reply:
locations is an object not an array